game/script.rpy

```diff
@@ -1070,49 +1068,47 @@ else:
 
     "I recognize the name Minos from a play in a town square about a labyrinth, years ago, but I don't recognize this specific story"
 
```

Comment:
I had gotten on board about the second version but some doubt has resurfaced. It feels like a bit of a throwaway compared to the other. Want to avoid the carpenter/merchant being "less learned and interested in Greek myth"